### PR TITLE
Add Multi-Chain IBC Transfers Example

### PR DESCRIPTION
**Pull request by bgodlin**

### CODE REVIEW

**Review by deverka — APPROVED** [review]
Looks good to me, but we also need @jamesbayly to have a look

**Comment by deverka** [comment]
@jamesbayly are you able to check this one?

File: Multi-Chain/osmosis-cosmos-bridge/README.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# SubQuery - Multi-Chain. Example project for IBC transfers between Osmosis and Cosmos Hub
+
+[SubQuery](https://subquery.network) is a fast, flexible, and reliable open-source data indexer that provides you with custom APIs for your web3 project across all of our supported networks. To learn about how to get started with SubQuery, [visit our docs](https://academy.subquery.network).
+
+**The SubQuery project is responsible for indexing all IBC transfer events occurring between Cosmos Hub and Osmosis**
```
Comment (jamesbayly) [inline Multi-Chain/osmosis-cosmos-bridge/README.md:5]:
```suggestion
**This SubQuery project indexes all IBC transfer events between Cosmos Hub and Osmosis**
```

File: Multi-Chain/osmosis-cosmos-bridge/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      DB_PORT: 5432
+    volumes:
+      - ./:/app
+    command:
+      - ${SUB_COMMAND:-} # set SUB_COMMAND env variable to "test" to run tests
+      - --multi-chain
+      - --db-schema=multi-transfers
+      - --disable-historical
+      - -f=app/cosmoshub.yaml
+      - --batch-size=1
```
Comment (jamesbayly) [inline Multi-Chain/osmosis-cosmos-bridge/docker-compose.yml:67]:
```suggestion
```
Don't commit this kind of batch-size

File: Multi-Chain/osmosis-cosmos-bridge/package.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "osmosis-cosmos-ibc",
+  "version": "0.0.1",
+  "description": "This project can be use as a starting point for developing your Cosmos (Cosmos Hub) based SubQuery project",
```
Comment (jamesbayly) [inline Multi-Chain/osmosis-cosmos-bridge/package.json:4]: Update to match the readme
```suggestion
  "description": "This SubQuery project indexes all IBC transfer events between Cosmos Hub and Osmosis",
```

File: Multi-Chain/osmosis-cosmos-bridge/schema.graphql [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# To improve query performance, we strongly suggest adding indexes to any field that you plan to filter or sort by
+# Add the `@index` or `@index(unique: true)` annotation after any non-key field
+# https://academy.subquery.network/build/graphql.html#indexing-by-non-primary-key-field
+
+type CosmosToOsmosisBridge @entity {
```
Comment (jamesbayly) [inline Multi-Chain/osmosis-cosmos-bridge/schema.graphql:5]: Can you Rename this simple to `BridgeEvent`
`CosmosToOsmosisBridge` kinda infers that this is one-directional between Cosmos -> Osmosis
Then in the future it's easy for us to add other networks to this project

File: Multi-Chain/osmosis-cosmos-bridge/src/mappings/mappingHandlers.ts [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+    bridgeTransactionRecord.destinationChainTransaction =
+      event.tx.hash.toString();
+  }
+  await bridgeTransactionRecord.save();
+}
+
+export async function handleOsmosisReceiveEvent(
+  event: CosmosEvent
+): Promise<void> {
+  logger.warn(
```
Comment (jamesbayly) [inline Multi-Chain/osmosis-cosmos-bridge/src/mappings/mappingHandlers.ts:98]: Instead of `logger.warn`, these should instead all be `logger.info`

**Review by jamesbayly — CHANGES_REQUESTED** [review]

**Pull request merged**